airbyte-integrations/connectors/destination-csv/src/main/java/io/airbyte/integrations/destination/csv/CsvDestination.java

```diff
@@ -87,7 +87,7 @@ public AirbyteConnectionStatus check(JsonNode config) {
   }
 
   @Override
-  public SQLNamingResolvable getNamingResolver() {
+  public IdentifierNamingResolvable getNamingResolver() {
```

Comment:
since your not doing any deps injection here, you should use the most precise type here (StandardNaming)